tests/format_sql/template_statement_example/expect.sql

```diff
@@ -0,0 +1,13 @@
+{% include "udf.sql" %}
+SELECT
+  {% for column in column_list %}
+    {{ column }},
```

Comment:
This would leave a trailing comma which wouldn't be valid SQL, correct?

You could use Jinja's `join` filter: `{{ column_list|join(", ") }}`.

Reply:
This is true, although this is a (semi-) contrived example for unit testing.

Reply:
Also: BigQuery allows trailing commas in the `SELECT` section